charger/easee.go

```diff
@@ -340,7 +336,14 @@ func (c *Easee) ChargerUpdate(i json.RawMessage) {
 
 // CommandResponse implements the signalr receiver
 func (c *Easee) CommandResponse(i json.RawMessage) {
-	// c.observe("CommandResponse", i)
+	var res easee.SignalRCommandResponse
+
+	err := json.Unmarshal(i, &res)
+	if err != nil {
```

Comment:
Das wird ja in der Realität nicht passieren- also einfach nur den err loggen. Die Message wird über den http request eh geloggt.